include/fire/io/Data.h

```diff
@@ -414,6 +504,8 @@ class Data<AtomicType, std::enable_if_t<is_atomic_v<AtomicType>>>
 template <typename ContentType>
 class Data<std::vector<ContentType>>
     : public AbstractData<std::vector<ContentType>> {
+  /// the version of serialization of this container type
+  using version = std::integral_constant<int,0>;
```

Comment:
Not sure if this will work out-of-the-box, but
```suggestion
    fire_class_version(1);
```